packages/account/src/Components/article/article.jsx

```diff
@@ -9,7 +10,9 @@ const Article = ({ className, title, descriptions }) => (
             {descriptions &&
                 descriptions.map((desc, idx) => (
                     <li key={idx}>
-                        <p className='account__article-text'>{desc}</p>
+                        <Text className='account__article-text' as='p'>
```

Comment:
I don't see any CSS changes in this PR. So the question is did you search for related class names related to text properties and remove those which are replaced by `Text`?